raster/src/main/scala/geotrellis/raster/io/geotiff/tags/CmykTags.scala

```diff
@@ -17,12 +17,16 @@
 package geotrellis.raster.io.geotiff.tags
 
 import monocle.macros.Lenses
+import io.circe.generic.JsonCodec
 
 @Lenses("_")
+@JsonCodec
 case class CmykTags(
   inkSet: Option[Int] = None,
   numberOfInks: Option[Int] = None,
   inkNames: Option[String] = None,
   dotRange: Option[Array[Int]] = None,
   targetPrinters: Option[String] = None
 )
+
+object CmykTags
```

Comment:
Hm, so if we've chosen a no `double` annotations path, then probably it should follow a usual style with manual codecs definition in the companion object.